sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetSchemaPruning.scala

```diff
@@ -81,7 +82,8 @@ object ParquetSchemaPruning extends Rule[LogicalPlan] {
    * Checks to see if the given relation is Parquet and can be pruned.
```

Comment:
`ParquetSchemaPruning` can be renamed to like `SchemaPruning`, but I leave it to later followup to reduce diff.

Reply:
After finishing reviews, we can rename it as a final commit.